### ISSUE TITLE
[Bug] `mr close` panics with index out of range [0] with length 0

### ISSUE THREAD

**Issue by profclems** [profclems/glab#301]
`glab mr close` panics when merge request ID is not provided.

**Expected Behaviour**

If the merge request id is not provided, `glab` should find the open merge request for the current checked out branch and close it

**Actual Behaviour**
```sh
→ glab mr close
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/profclems/glab/commands/mr/close.NewCmdClose.func1(0xc0002e8580, 0x14a4e48, 0x0, 0x0, 0x0, 0x0)
        github.com/profclems/glab/commands/mr/close/mr_close.go:43 +0x605
github.com/spf13/cobra.(*Command).execute(0xc0002e8580, 0x14a4e48, 0x0, 0x0, 0xc0002e8580, 0x14a4e48)
        github.com/spf13/cobra@v1.0.0/command.go:842 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000405080, 0xc000102160, 0x2, 0x2)
        github.com/spf13/cobra@v1.0.0/command.go:950 +0x375
main.main()
        github.com/profclems/glab/cmd/glab/main.go:114 +0x736
```

It works when merge request ID is provided
```sh
→ glab mr close 359
- Closing Merge request...✔ Merge request !359
!359 fix: error 500 when creating purchase order' (fix-err-500-purchase-order)
 https://gitlab.com/ssisgh-projects/simeds/back-end/-/merge_requests/359
```

**Comment by zemzale** [comment]
I started to implement a fix for this, but came up to some stuff that I found a bit weird. 
In `commands/mr/close/mr_close.go` we are getting the current branches merge request ([reference](https://github.com/profclems/glab/blob/bc7bafcc0538d2490f8a6181f464439bb3caf3ca/commands/mr/close/mr_close.go#L29)).
Later on we are ignoring this and just getting MR ID from args where the panic is thrown. 

Then the code tries to split that argument string into multiple MR IDs which then would allow to close multiple merge requests.
The problem is that this can't happen because if the getting of MR from arguments fails the command just fails.
Example: 
```sh
glab mr close 20,19
no open merge request available for "20,19"
```

My question, do we just remove the not working ability to close multiple merge requests, or we fix that while also fixing this panic?
